docker/testCaseD/ssh_config

```diff
@@ -0,0 +1,19 @@
+Host hostc
```

Comment:
i thought this file is autogenerated and therefore has no need to be committed?

Reply:
``docker/testCaseD`` is only there for Documentation purposes.
This will be refactored to allow easy functional tests.